3.6/release-notes-new-features36.md

```diff
@@ -12,12 +12,128 @@ here.
 ArangoSearch
 ------------
 
-ArangoSearch views are now eligible for SmartJoins in AQL, provided that their
+### SmartJoins and Views
+
+ArangoSearch Views are now eligible for SmartJoins in AQL, provided that their
 underlying collections are eligible too.
 
+### Dynamic search expressions with arrays
+
+ArangoSearch now accepts expressions with array comparison operators in the
+form of:
+
+```
+<array> [ <ALL|ANY|NONE> ] [ <=|<|==|!=|>|>=|IN ] doc.<attribute>
+```
+
+i.e. the left-hand side operand is always an array, which can be dynamic.
+
+```js
+LET tokens = TOKENS("some input", "text_en")
+FOR doc IN myView SEARCH tokens ALL IN doc.title RETURN d   // dynamic conjunction
+FOR doc IN myView SEARCH tokens ANY IN doc.title RETURN d   // dynamic disjunction
+FOR doc IN myView SEARCH tokens NONE IN doc.title RETURN d  // dynamic negation
+FOR doc IN myView SEARCH tokens ALL > doc.title RETURN d    // dynamic conjunction with comparison
+FOR doc IN myView SEARCH tokens ANY <= doc.title RETURN d   // dynamic disjunction with comparison
+```
+
+In addition, both the `TOKENS()` and the `PHRASE()` functions were
+extended with array support.
+
+`TOKENS()` accepts recursive arrays of strings as the first argument:
+
+```js
+RETURN TOKENS("quick brown fox", "text_en")        // returns [ "quick", "brown", "fox" ]
+RETURN TOKENS(["quick brown", "fox"], "text_en")   // returns [ ["quick brown"], ["fox"] ]
+RETURN TOKENS(["quick brown", ["fox"]], "text_en") // returns [ ["quick brown"], [["fox"]] ]
+```
+
+`PHRASE()` accepts an array as the second argument:
+
+```js
+LET tokens = TOKENS("quick brown fox", "text_en") // ["quick", "brown", "fox"]
+FOR doc IN myView SEARCH PHRASE(doc.title, tokens, "text_en") RETURN d
+```
+
+### Analyzers
+
+Added UTF-8 support and ability to mark beginning/end of the sequence to `ngram` Analyzer
+
+The following optional properties can be provided with `ngram` analyzer definition:
+
+- `startMarker` : `<string>`, default: ""<br>
+  this value will be prepended to ngrams at the beginning of input sequence
+
+- `endMarker` : `<string>`, default: ""<br>
+  this value will be appended to ngrams at the beginning of input sequence
+
+- `streamType` : `"binary"|"utf8"`, default: "binary"<br>
+  type of the input stream
+
+### Added "Edge ngram" support to `text` analyzer
+
+The following optional properties can be provided with `text`
+Analyzer definition:
```

Comment:
```suggestion
analyzer definition:
```

Reply:
I consider Analyzers a feature name and capitalized the first letter in most of the documentation, so this should stay as-is.

Reply:
Then it should be capitalized consistently. I think it isn't right now.

Reply:
Should be fixed by https://github.com/arangodb/docs/pull/235/commits/12f6cd12c04d17c6fee4bcd494c73089739028b2

The remaining occurrences should refer to a variable:

```
const analyzers = await db.listAnalyzers();
// analyzers is an array of Analyzer descriptions
```